.github/workflows/DocPreviewCleanup.yml

```diff
@@ -0,0 +1,33 @@
+name: Doc Preview Cleanup
+
+on:
+  pull_request:
+    types: [closed]
```

Comment:
I have little experience with GitHub Actions, so pardon any ignorant questions on my part. What does `types: [closed]` indicate? Does it run for all PR's, including drafts?

Reply:
This new action does not actually perform the preview upload. That is still done by the documenter action from before (now it provides the preview because of the `push_preview=true` keyword in `deploydocs`). This new action only cleans up the GitHub repo by deleting the previews again once they are not needed anymore (i.e. once the corresponding PR is closed). This is only necessary to avoid cluttering the gh-pages branch of the repo where all of the docs previews will be uploaded to.

Reply:
Ah! That makes sense. Looks like I interpreted this line correctly but missed the context for the action itself